added marc as a member (#4)
pivlab/pivlab-website: _members/marc-subirana.md

---
name: Marc Subirana Granés
image: images/team/marc-subirana.jpg
description: Postdoc
role: postdoc
links:
  email: marc.subiranagranes@cuanschutz.edu
  github: msubirana
---

> Marc is currently a Postdoctoral Researcher at [Pividori Lab](/) in the [Department of Biomedical Informatics](https://medschool.cuanschutz.edu/dbmi) and the [Center for Health Artificial Intelligence (AI)](https://medschool.cuanschutz.edu/ai) at the [University of Colorado Anschutz Medical Campus](https://www.cuanschutz.edu/).

Reviewer: ```suggestion
Marc is currently a Postdoctoral Researcher at [Pividori Lab](/) in the [Department of Biomedical Informatics](https://medschool.cuanschutz.edu/dbmi) and the [Colorado Center for Personalized Medicine (CCPM)](https://medschool.cuanschutz.edu/ccpm) at the [University of Colorado Anschutz Medical Campus](https://www.cuanschutz.edu/).
```

> Prior to joining the Pividori Lab, Marc earned his PhD in Biomedicine and Bioinformatics at [Endocrine regulatory genomics](https://www.upf.edu/web/endocrine-regulatory-genomics/research) at the [Parcelona Biomedical Research Park](https://www.prbb.org/) (Barcelona, Spain). His primary research focused on insulinomas, a form of pancreatic cancer, where he analyzed the associated regulatory networks and somatic mutations. This work led to the discovery of genetic variations that affect the deregulation of noncoding functions in insulinoma. Furthermore, he extended his research to study the regulatory networks in type 1 diabetes, aiming to better understand the disease's mechanisms. Beyond his research, he developed [UMI4Cats](https://bioconductor.org/packages/release/bioc/html/UMI4Cats.html), an efficient, user-friendly software for analyzing quantitative chromatin contacts, now widely available through Bioconductor. His contributions have been pivotal in advancing our understanding of complex diseases and enhancing bioinformatics tools in the scientific community.

Reviewer: I suggest one line per sentence to simplify reviewing.

```suggestion
Prior to joining the Pividori Lab, Marc earned his PhD in Biomedicine and Bioinformatics at [Endocrine regulatory genomics](https://www.upf.edu/web/endocrine-regulatory-genomics/research) at the [Parcelona Biomedical Research Park](https://www.prbb.org/) (Barcelona, Spain).
His primary research focused on insulinomas, a form of pancreatic cancer, where he analyzed the associated regulatory networks and somatic mutations.
This work led to the discovery of genetic variations that affect the deregulation of noncoding functions in insulinoma.
Furthermore, he extended his research to study the regulatory networks in type 1 diabetes, aiming to better understand the disease's mechanisms.
Beyond his research, he developed [UMI4Cats](https://bioconductor.org/packages/release/bioc/html/UMI4Cats.html), an efficient, user-friendly software for analyzing quantitative chromatin contacts, now widely available through Bioconductor.
His contributions have been pivotal in advancing our understanding of complex diseases and enhancing bioinformatics tools in the scientific community.
```

> As a Postdoctoral Researcher in the Pividori Lab, Marc is focused on exploiting next generation of computational methods and artificial intelligence methods to dissect genetic and epigenetic data. His goal is to extract meaningful biological insights that can pave the way for novel interventions and therapies, ultimately contributing to the improvement of human health.

Reviewer: Applying one line per sentence.

```suggestion
As a Postdoctoral Researcher in the Pividori Lab, Marc is focused on exploiting next generation of computational methods and artificial intelligence methods to dissect genetic and epigenetic data.
His goal is to extract meaningful biological insights that can pave the way for novel interventions and therapies, ultimately contributing to the improvement of human health.
```

> Outside of his professional endeavors, Marc is passionate about outdoor activities, including hiking, biking, climbing, and skiing. He values quality time with friends and family and enjoys organizing large dinners to celebrate these connections.

Reviewer: Applying one line per sentence.

```suggestion
Outside of his professional endeavors, Marc is passionate about outdoor activities, including hiking, biking, climbing, and skiing.
He values quality time with friends and family and enjoys organizing large dinners to celebrate these connections.
```
